spec/definitions.yaml

```diff
@@ -183,7 +183,7 @@ definitions:
         description: |
           Number of availability zones a cluster should be spread across. The
```

Comment:
Should this description change? It's just the number of AZ's on AWS, but it's the actual AZ's to use on Azure.

Reply:
Good catch!